src/components/ButtonList/index.js

```diff
@@ -14,4 +14,5 @@ const ButtonList = ({ currentIndex, list, stateNumb }) =>
       {index + 1 >= 10 ? index + 1 : "0" + (index + 1)}
```

Comment:
you have helpers in another page for this
maybe you use utils (name it formatterIndex)